nix/default.nix

```diff
@@ -251,10 +255,43 @@ let
       '';
     };
 
+  buildOgmiosFixtures =
```

Comment:
As I mentioned in another comment, this can just be moved to the flake or another module since it's only useful for CTL itself most likely